src/OpenTelemetry.Abstractions/Trace/Event.cs

```diff
@@ -55,7 +55,7 @@ public static IEvent Create(string name)
         /// Returns a new <see cref="Event"/> with the provided name and set of attributes.
         /// </summary>
         /// <param name="name">The text name for the <see cref="Event"/>.</param>
-        /// <param name="attributes">The <see cref="IDictionary{string, IAttributeValue}"/> of attributes for the <see cref="Event"/>.</param>
+        /// <param name="attributes">The <see cref="IDictionary{String, IAttributeValue}"/> of attributes for the <see cref="Event"/>.</param>
```

Comment:
nit: why `String` is needed?

Reply:
I believe it was giving a warning otherwise as doc parser didn't find the type =)

Reply:
The change is for **Warning CS1658 Type parameter declaration must be an identifier not a type. See also error CS0081**.

Reply:
got it, thanks for the explanation!